glue/plugins/wcs_autolinking/wcs_autolinking.py

```diff
@@ -12,21 +14,45 @@ class IncompatibleWCS(Exception):
 
 def get_cids_and_functions(wcs1, wcs2, pixel_cids1, pixel_cids2):
 
-    def forwards(*pixel_input):
-        return pixel_to_pixel(wcs1, wcs2, *pixel_input)
+    if len(pixel_cids1) != 1:
+        def forwards(*pixel_input):
+            return pixel_to_pixel(wcs1, wcs2, *pixel_input)
 
-    def backwards(*pixel_input):
-        return pixel_to_pixel(wcs2, wcs1, *pixel_input)
+        def backwards(*pixel_input):
+            return pixel_to_pixel(wcs2, wcs1, *pixel_input)
 
-    pixel_input = (0,) * len(pixel_cids1)
+        pixel_input = (0,) * len(pixel_cids1)
 
-    try:
-        forwards(*pixel_input)
-        backwards(*pixel_input)
-    except Exception:
-        return None, None, None, None
+        if len(pixel_cids1) == 1:
+            pixel_input = np.zeros((1,1))
 
-    return pixel_cids1, pixel_cids2, forwards, backwards
+        try:
+            forwards(*pixel_input)
+            backwards(*pixel_input)
+        except Exception:
+            return None, None, None, None
+
+        return pixel_cids1, pixel_cids2, forwards, backwards
+
+    else:
+        def forwards(*pixel_input):
+            return wcs2.world_to_pixel(*wcs1.pixel_to_world(pixel_input))
```

Comment:
Why does pixel_to_pixel not work here?

Reply:
Not sure, but it wasn't working before, hence the changes introduced...

Reply:
Could you take a quick look again and let me know what errors you get if you do just use the default forwards and backwards?

Reply:
The error is gone now... 🤔